### PR TITLE
Reconfiguring EcoBridge

### PR DESCRIPTION
**Pull request by bejzik8**
# Summary

This pull request reconfigures `EcoBridge`, adds bridging support between `Arbitrum Goerli` and `Goerli`, and removes `Rinkeby`. It also enables test networks when the application runs locally.

🍀

### CODE REVIEW

**Comment by bejzik8** [comment]
Can someone take a look at this?

**Review by Mi-Lan — APPROVED** [review]
Nice job!

File: src/pages/Bridge/index.tsx [hunk 1]
```diff
@@ -296,7 +296,7 @@ export default function Bridge() {
         onNetworkChange: onFromNetworkChange,
         selectedNetworkChainId: isCollecting && collectableTx ? collectableTx.fromChainId : fromChainId,
         activeChainId: account ? chainId : -1,
-        showTestnets: false,
```
Comment (berteotti) [inline src/pages/Bridge/index.tsx:299]: It's better to use `const { isDevelopment } = useEnvironment()` here

**Review by berteotti — APPROVED** [review]
Looks fine, just made a small comment

**Pull request merged**